package.json

```diff
@@ -21,12 +21,16 @@
 		"@storybook/addon-postcss": "^2.0.0",
 		"@testing-library/jest-dom": "^5.16.5",
 		"@testing-library/react": "^14.0.0",
+		"@testing-library/user-event": "^14.4.3",
 		"axios": "^1.3.5",
+		"jsdoc": "^4.0.2",
 		"jsdom": "^21.1.1",
 		"react": "^18.2.0",
 		"react-dom": "^18.2.0",
 		"react-redux": "^8.0.5",
 		"react-router-dom": "^6.10.0",
+		"react-spinners": "^0.13.8",
```

Comment:
You can remove this package. Let's only use packages for things we can't implement ourselves.